05/css/mobile.css

```diff
@@ -0,0 +1,158 @@
+@media screen and (max-width: 800px) {
+  .content {
+    display: flex;
+    flex-direction: column;
+    margin: 2px;
+  }
+
+  /* Header */
+  .header {
+    flex-direction: row;
+    justify-content: space-between;
+  }
```

Comment:
Zrezygnowałbym z tych ustawień:

![image](https://github.com/devmentor-pl/practice-html-and-css-rwd/assets/55095257/ed43c8e9-7732-4429-b74d-0ad03b8c5468)